pkg/limayaml/validate.go

```diff
@@ -159,6 +161,7 @@ func Validate(y *LimaYAML, warn bool) error {
 			if !errors.Is(err, os.ErrNotExist) {
 				return fmt.Errorf("field `mounts[%d].location` refers to an inaccessible path: %q: %w", i, f.Location, err)
 			}
+			mountWarnings = append(mountWarnings, fmt.Sprintf("field `mounts[%d].location` refers to a non-existent directory: %q:", i, f.Location))
```

Comment:
This line should just be

```suggestion
			logrus.Warnf("field `mounts[%d].location` refers to a non-existent directory: %q:", i, f.Location)
```

However, I wonder if we shouldn't just treat any error to `stat` the mount location as an error? @AkihiroSuda WDYT?

Reply:
> However, I wonder if we shouldn't just treat any error to stat the mount location as an error?

Why?

Reply:
How does it make sense to mount a directory that does not exist?

If `stat` succeeds, but the entry is not a directory, we throw an error. How is a non-existing entry any different: it also is not a directory.